bench_method.go

```diff
@@ -38,7 +39,7 @@ type benchmarkMethod struct {
 // up by making some number of requests through it.
 func (m benchmarkMethod) WarmTransport(opts TransportOptions, warmupRequests int) (transport.Transport, error) {
 	opts.benchmarking = true
-	transport, err := getTransport(opts, m.serializer.Encoding())
+	transport, err := getTransport(opts, m.serializer.Encoding(), opentracing.NoopTracer{})
```

Comment:
Just realized, baggage only gets with the Jaeger tracer right? Perhaps we can't do this unless we explicitly disallow baggage with benchmarking? Otherwise, the initial request will use baggage but the benchmarking requests will not.